src/EditorFeatures/Core/InheritanceMargin/InheritanceMarginOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.InheritanceMargin
+{
+    internal class InheritanceMarginOptions
```

Comment:
❔ Is there a corresponding `IOptionsProvider` for this?

Reply:
Moved to FeatureOnOffOptions